pkg/controller/installation/products/rhsso/reconciler.go

```diff
@@ -3,55 +3,62 @@ package rhsso
 import (
 	"context"
 	"fmt"
-	"strings"
-
-	v1alpha12 "github.com/integr8ly/integreatly-operator/pkg/apis/monitoring/v1alpha1"
-	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/monitoring"
-
-	aerogearv1 "github.com/integr8ly/integreatly-operator/pkg/apis/aerogear/v1alpha1"
 	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	v1alpha12 "github.com/integr8ly/integreatly-operator/pkg/apis/monitoring/v1alpha1"
 	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/marketplace"
 	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/config"
+	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/monitoring"
 	"github.com/integr8ly/integreatly-operator/pkg/resources"
+	keycloak "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
 	appsv1 "github.com/openshift/api/apps/v1"
-	oauthv1 "github.com/openshift/api/oauth/v1"
+	v12 "github.com/openshift/api/route/v1"
 	usersv1 "github.com/openshift/api/user/v1"
 	oauthClient "github.com/openshift/client-go/oauth/clientset/versioned/typed/oauth/v1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/ownerutil"
 	"github.com/pkg/errors"
-	pkgerr "github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
-	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	errors2 "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	pkgclient "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"strings"
 )
 
 var (
 	defaultRhssoNamespace               = "rhsso"
 	customerAdminPassword               = "Password1"
 	keycloakName                        = "rhsso"
 	keycloakRealmName                   = "openshift"
-	defaultSubscriptionName             = "integreatly-rhsso"
+	defaultSubscriptionName             = "keycloak-rhsso"
 	idpAlias                            = "openshift-v4"
 	githubIdpAlias                      = "github"
 	githubOauthAppCredentialsSecretName = "github-oauth-secret"
-	manifestPackage                     = "integreatly-rhsso"
+	manifestPackage                     = "keycloak-rhsso"
+)
+
+const (
+	SSOLabelKey   = "sso"
+	SSOLabelValue = "integreatly"
+	RHSSOProfile  = "RHSSO"
 )
 
-var CustomerAdminUser = &aerogearv1.KeycloakUser{
-	KeycloakApiUser: &aerogearv1.KeycloakApiUser{
-		Enabled:       true,
-		Attributes:    aerogearv1.KeycloakAttributes{},
-		UserName:      "customer-admin",
-		EmailVerified: true,
-		Email:         "customer-admin@example.com",
-		ClientRoles:   getKeycloakRoles(true),
+var CustomerAdminUser = keycloak.KeycloakAPIUser{
```

Comment:
We don't need to create this user anymore.
This user was added in original implementation, but since then we started using Groups, and this user is no longer needed.

Reply:
@matskiv Has this user creation already been removed from master?

Reply:
@davidffrench no. I wanted to do that, but removing it in this PR seemed more efficient.

Reply:
@matskiv Ok, perfect. My preference would be a separate PR which would make sense after this PR is merged. Which may be 2021 :-D

Reply:
@davidffrench okay, that's fine :)
Let me know once this is ready for a review. I can then review and create follow up JIRAs for any changes(such as removing customer-admin) that will be required.

Reply:
Sounds good to me, thanks @matskiv

Reply:
@matskiv Just to ensure this is not missed @matskiv , can you create a JIRA for removing customer-admin if you haven't already and link it in JIRA as a follow on task from https://issues.redhat.com/browse/INTLY-3860